requirements-dev.txt

```diff
@@ -0,0 +1,5 @@
+-r requirements.txt
```

Comment:
I think the behaviour described in Jira ticket is different:

- requirements-dev.txt contains only development dependencies that are currently in requirements.txt, for example pylint, pytest etc.
- then requirements.txt contains all -r lines referencing other files

Reply:
I think there is no need to create requirements-dev.txt then, the -r requirements files can be moved to the requirements.txt files and when is -dev switch called on the new script, only requirements.txt files will get updated (check pur --no-recursive).